v3.0/reference/sql/statements/split-region.md

```diff
@@ -160,7 +160,7 @@ SPLIT TABLE t INDEX idx3 BETWEEN ("2010-01-01 00:00:00", "a") AND ("2010-01-01 0
 {{< copyable "sql" >}}
 
 ```sql
-SPLIT TABLE t1 INDEX idx4 ("a", "2000-01-01 00:00:01"), ("b", "2019-04-17 14:26:19"), ("c", "");  
+SPLIT TABLE t1 INDEX idx4 BY ("a", "2000-01-01 00:00:01"), ("b", "2019-04-17 14:26:19"), ("c", "");
```

Comment:
Currently, the `dev` folder should keep consistent with the `v3.0` folder. So if you have any updates in one of the folders,  please remember updating the other one too. Thanks!

Reply:
done